.clang-format

```diff
@@ -2,7 +2,7 @@
 Language:        Cpp
 # BasedOnStyle:  LLVM
 AccessModifierOffset: -2
-AlignAfterOpenBracket: Align
+AlignAfterOpenBracket: true
```

Comment:
This fixes an error about an unsupported value for this choice. Without it clang-format reformats close to the entire code-base for me. Without, I only get a format of my own files.

Will open a dedicated PR for it.